src/StoreKeeper/WooCommerce/B2C/Frontend/Handlers/AddressFormHandler.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+namespace StoreKeeper\WooCommerce\B2C\Frontend\Handlers;
+
+use StoreKeeper\WooCommerce\B2C\Endpoints\EndpointLoader;
+use StoreKeeper\WooCommerce\B2C\Endpoints\WebService\AddressSearchEndpoint;
+use StoreKeeper\WooCommerce\B2C\I18N;
+use StoreKeeper\WooCommerce\B2C\Options\AbstractOptions;
+
+class AddressFormHandler
+{
+    public const STREET_ADDRESS_POSITION = 4;
+    public const HOUSE_NUMBER_FIELD = 'address_house_number';
```

Comment:
Did you test if this field is save to StoreKeper on the order?

Reply:
Added house number to `OrderExport` if it exists. Verified working.
![image](https://user-images.githubusercontent.com/18332309/150110115-ff16c0bf-6338-460b-80a2-ba4885c4daad.png)